silverback/cluster/types.py

```diff
@@ -42,25 +48,27 @@ class ClusterConfiguration(BaseModel):
     memory: Annotated[int, Field(ge=0, le=120)] = 0  # defaults to 512 MiB
     """Total memory per bot (in GB, 0 means '512 MiB')"""
 
-    # NOTE: Configure # of workers based on cpu & memory settings
+    # NOTE: # of workers configured based on cpu & memory settings
 
-    # Runner configuration (Bytes 3-5)
+    # Runner configuration (Bytes 3-4)
     networks: Annotated[int, Field(ge=1, le=20)] = 1
     """Maximum number of concurrent network runners"""
 
     bots: Annotated[int, Field(ge=1, le=250)] = 1
-    """Maximum number of concurrent bots running"""
+    """Maximum number of concurrent running bots"""
 
-    triggers: Annotated[int, Field(ge=50, le=1000, multiple_of=5)] = 50
-    """Maximum number of task triggers across all running bots"""
+    # NOTE: Byte 5 unused
```

Comment:
question: got something in mind?

Reply:
not right now, reserved is maybe better?
```suggestion
    # NOTE: Byte 5 reserved
```

Reply:
Language is whatever. I was just curious if there was a plan. The fact that it's right in the middle seemed intentional.

Reply:
It's more that I wanted to remove `triggers` and didn't want to re-arrange the order of the integers